lib/workers/repository/init/apis.js

```diff
@@ -7,6 +8,18 @@ function assignPlatform(config) {
   return config;
 }
 
+async function setGitPrivateKey(config) {
```

Comment:
I'd like to move this initialization code out of workers/repository and into gitFs's initRepo function: https://github.com/renovatebot/renovate/blob/14214835a116d79007d7dc2bd3f4913eb73cdbea/lib/platform/git/storage.js#L53